### ISSUE TITLE
Composer helper : allow run as other user

### ISSUE THREAD

**Issue by Salamandar** [YunoHost/issues#2367]
### Describe the bug

```
# ynh_composer_exec
[...]
Do not run Composer as root/super user! See https://getcomposer.org/root for details
Aborting as no plugin should be loaded if running as super user is not explicitly allowed

# ynh_exec_as "$app" ynh_composer_exec
sudo: ynh_composer_exec: command not found
```

**Comment by tituspijean** [comment]
I think I had at some point coded an alternative composer helper for Flarum with that fearure. We could reuse that.

**Comment by OniriCorpe** [comment]
maybe run it as `$app` by default or mandatory?

**Comment by alexAubin** [comment]
Somewhat fixed by helpers 2.1 which now runs composer as $app
